torch/onnx/symbolic_opset9.py

```diff
@@ -901,6 +901,29 @@ def batch_norm(g, input, weight, bias, running_mean, running_var, training, mome
         return res
 
 
+@parse_args('v', 'is', 'v', 'v', 'f', 'i')
+def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
+    axes = [-i for i in range(len(normalized_shape), 0, -1)]
+
+    two_cst = g.op("Constant", value_t=torch.tensor(2.))
+    eps_cst = g.op("Constant", value_t=torch.tensor(eps))
+
+    mean = g.op("ReduceMean", input, axes_i=axes)
+    squared_input = pow(g, input, two_cst)
+    squared_mean = pow(g, mean, two_cst)
+    squared_input_mean = g.op("ReduceMean", squared_input, axes_i=axes)
```

Comment:
Question: Could this variance computation be done using ONNX::std? If yes, is that preferable in anyway to this subgraph?

Reply:
yes, var=std^2.